Mathlib/Topology/Algebra/Module/LinearMap.lean

```diff
@@ -1021,7 +1021,7 @@ def restrictScalars (f : M →L[A] M₂) : M →L[R] M₂ :=
 
 variable {R}
 
-@[simp] -- @[norm_cast] -- Porting note: This theorem can't be a `norm_cast` theorem.
+@[simp]
```

Comment:
I'm not so sure about this one. The problem is that, to my surprise, `LinearMap.restrictScalars` has the `@[coe]` attribute (but `ContinuousLinearMap.restrictScalars` does not). So again, `norm_cast` is making the correct judgment here, and it's indicating a discrepancy in our handling of `restrictScalars` in the continuous case.

So for right now, would you mind reverting it?

Reply:
asked on [Zulip](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/.60LinearMap.2ErestrictScalars.60.20is.20a.20coercion.3F)

Reply:
Reverted.